Quaver.Shared/Skinning/SkinKeys.cs

```diff
@@ -543,8 +547,16 @@ private void LoadLaneSpecificElements()
                     ColumnColors[i] = ConfigHelper.ReadColor(ColumnColors[i], Store.Config[ModeHelper.ToShortHand(Mode).ToUpper()][$"ColumnColor{i + 1}"]);
 
                 // HitObjects
-                LoadHitObjects(NoteHitObjects, $"note-hitobject-{i + 1}", i);
-                LoadHitObjects(NoteHoldHitObjects, $"note-holdhitobject-{i + 1}", i);
+                if (UseSameHitObjects == true)
+                {
```

Comment:
As per the formatting guide for contributing, can you get rid of the brackets for these two IF statements.